src/templates/partials/caddy.hbs

```diff
@@ -1,43 +1,31 @@
 # {{output.header}}
 # {{{output.link}}}
-{{#unless (includes "old" form.config)}}
+{{#unless (minver "2.0.0" form.serverVersion)}}
+# note that Caddy version 1 reached end-of-life > 4 years ago back in 2020
+{{else}}
+{{#if (includes "old" form.config)}}
+# note that Caddy supports only TLSv1.2 and later
+{{else}}
 # note that Caddy automatically configures safe TLS settings
+{{/if}}
 {{/unless}}
 
 # replace example.com with your domain name
-example.com
-{{! This is a big of a kludge due to Caddy restrictions on TLS versions and cipher suites }}
-{{#if (includes "old" form.config)}}
-
-tls {
-    protocols tls1.0 tls1.3
-    {{#if (minver "2.0.0" form.serverVersion)}}
-    ciphers {{{join output.ciphers " "}}}
-    {{ else }}
-    ciphers ECDHE-ECDSA-AES128-GCM-SHA256 ECDHE-RSA-AES128-GCM-SHA256 ECDHE-ECDSA-AES256-GCM-SHA384 ECDHE-RSA-AES256-GCM-SHA384 ECDHE-ECDSA-WITH-CHACHA20-POLY1305 ECDHE-RSA-WITH-CHACHA20-POLY1305 ECDHE-ECDSA-AES128-CBC-SHA ECDHE-RSA-AES256-CBC-SHA ECDHE-RSA-AES128-CBC-SHA ECDHE-ECDSA-AES256-CBC-SHA RSA-AES128-CBC-SHA RSA-AES256-CBC-SHA RSA-3DES-EDE-CBC-SHA
-    {{/if}}
-}
-{{/if}}
-{{#if (includes "intermediate" form.config)}}
+example.com {
```

Comment:
Do I read it correctly that the config was not scoping the settings to the hostname before? Is that some recent change, or a preference in style, or the configs were rendered completely broken?

Reply:
was broooooken

Reply:
(I know the TLSv flags were completely wrong etc., I was was working on a fix to both v1 and v2 before tracing back the v1 configs only to v5.4 specs and nothing more etc. some time ago… but never really noticed the whole config was missing the actual closure!…)

I may want to come back to this later, but if it's super broken, the same case… feel free to publish self-r+ and will do a proper review later, following up with any changes separately.

Reply:
We should consider applying that logic to this whole site.  IMNSHO, we should be providing good advice for software released in the last 3 years, and should prioritize that much higher than pedantic historical curiosities for ancient configs.

I feel that there is no good reason to spend time updating the site for EOL software versions (caddy v1) or specs prior to guidelines 5.7 before the much higher priority of updating the site to provide good advice for contemporary software versions.

Per @janbrasna comment above, I will merge this as-is.

Please do feel free to revisit and improve.  Also, open invitation to other contributors, too!  @francislavoie, if nothing else, feedback will be more timely on PRs with @janbrasna and myself as new contributors to this github repo.